pkg/catalog/traffictarget_test.go

```diff
@@ -870,156 +870,3 @@ func TestListInboundTrafficTargetsWithRoutes(t *testing.T) {
 		})
 	}
 }
-
```

Comment:
Is it true that this test was defacto moved from `pkg/catalog/traffictarget_test.go` to `pkg/smi/client_test.go` ?